Spatial.gd

```diff
@@ -729,9 +744,14 @@ func on_gizmo_deselected(object):
 	$Control/Dialogs/NodeEditorDialog/ScrollContainer/VBoxContainer/SetActivePath.disabled = true
 	$Control/Dialogs/NodeEditorDialog/ScrollContainer/VBoxContainer/ReversePathDirection.disabled = true
 
-func on_update_made():
-	self.unsaved_changes = true
-	update_burrito_icon()
+func on_point_updated(position: Vector3):
+	set_unsaved_changes(true)
+	if self.currently_selected_node.point_type == "icon":
```

Comment:
Can we have logic that does not rely on `point_type` branching in multiple places?